packages/components/src/color-picker/color-copy-button.tsx

```diff
@@ -56,20 +55,17 @@ export const ColorCopyButton = ( props: ColorCopyButtonProps ) => {
 
 	return (
 		<Tooltip
-			content={
-				<Text color="white">
-					{ copiedColor === color.toHex()
-						? __( 'Copied!' )
-						: __( 'Copy' ) }
-				</Text>
+			delay={ 0 }
+			hideOnClick={ false }
+			text={
+				copiedColor === color.toHex() ? __( 'Copied!' ) : __( 'Copy' )
 			}
-			placement="bottom"
 		>
 			<CopyButton
-				isSmall
 				ref={ copyRef }
 				icon={ copy }
 				showTooltip={ false }
+				size="small"
```

Comment:
Is this change necessary and/or related to the tooltip swap?

Reply:
This isn't related to tooltip, so I can remove it. I saw the deprecation warning and thought I could update it while I was already there.

Reply:
Ah, I had missed the `isSmall` prop. Let's leave it as-is for this PR, and then we can open a PR to refactor all instances of `isSmall` to `size="small"` ?